mirrormanager2/lib/hostconfig.py

```diff
@@ -25,9 +25,14 @@
 
 import mirrormanager2.lib
 
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
```

Comment:
I believe that you can set the root logger's log level, and all child loggers will inherit it. Something like this should do it:

```
import logging

root_logger = logging.getLogger()
root_logger.setLevel(logging.INFO)
```

I can't remember whether it's important for that statement to happen earlier than all other statements, or if it can happen at any time. In Pulp there is a similar statement very early during process startup, and all the loggers inherit that logger's log level (and handler too). I wrote most of the code in Pulp's logging setup, and you can see it here:

https://github.com/pulp/pulp/blob/pulp-2.8.4-1/server/pulp/server/logs.py

Line 53 is where the root logger gets set up.